kwc-challenge-set.html

```diff
@@ -8,27 +12,469 @@
+                stroke-width: 4px;
+            }
+            image.unlocked {
+                fill: var(--color-kano-orange);
+                stroke: var(--color-kano-orange);
+            }
+            .circle-box {
+                padding-right: 15px;
+            }
+            .control-box {
+                padding-left: 15px;
+            }
+            .circle-content {
+                position: relative;
+            }
+            .control-subtitle,
+            .control-complete {
+                font-size: 16px;
+                font-weight: bold;
+                color: var(--color-grey);
+                margin: 0px;
+                padding: 0;
+                text-transform: uppercase;
+            }
+            .control-subtitle {
+                color: var(--color-grey);
+            }
+            .control-complete {
+                color: var(--color-kano-orange);
+            }
+            .control-content {
+                font-size: 16px;
+                color: var(--color-grey);
+                margin: 0px;
+                padding: 20px 0;
+            }
+            .control-title {
+                font-weight: bold;
+                color: var(--color-abbey);
+                font-family: var(--font-body);
+                font-size: 20px;
+                margin: 0;
+            }
+            .control-box {
+                @apply --layout-flex;
+                border-radius: 6px;
+                @apply --layout-vertical;
+                @apply --layout-center-justified;
+                align-items: flex-start;
+            }
+            .circle-box iron-image {
+                width: 120px;
+                height: 132px;
+                position: absolute;
+                @apply --layout-fit;
+                margin: auto;
+            }
+            .control-button {
+                margin: 40px 0 0 0;
+            }
+            [hidden] {
+                display:none !important;
+            }
+        </style>
+        <div id="container" hidden$="[[!_hasData(challengeSet)]]">
+
+            <div class="circle-box">
+                <div class="circle-content">
+                    <div id="canvas"></div>
+                    <div id="progress"></div>
+                    <iron-image src="[[_computeCompleteImage(_completed)]]"></iron-image>
+                </div>
+            </div>
+
+            <div class="control-box">
+                <p class="control-title">
+                    [[challengeSet.title]]
+                </p>
+                <p class$="[[_computeCompleteClass(_completed)]]">
+                    [[_computeProgressText(progress, _completed)]]
+                </p>
+                <p class="control-content">
+                    [[challengeSet.description]]
+                </p>
+                <kwc-button class="control-button"
+                    hidden$="[[_completed]]"
+                    on-tap="_openNextChallenge"
+                >
```

Comment:
Hello, here is the "indentation and arbitrary code standards" police and I have noticed you could have placed this `<` (and a few others around) on the previous line. I am fixing it for you this time because I am cool but if it was my boss, oh boy, you would be in trouble.

Reply:
Oh well, I can't change this file... So, could you? 👍

Reply:
I left this one as it makes it clearer that it contains text as a child